package.json

```diff
@@ -23,5 +24,9 @@
     "eslint-plugin-react-hooks": "^4.6.2",
     "eslint-plugin-react-refresh": "^0.4.7",
     "vite": "^5.3.4"
-  }
+  },
+  "description": "Make sure to change the .env file to point to the right\r backend. Then run the frontend using in development mode\r using:",
```

Comment:
Please delete the description, main and author lines. They are either wrong or unnecessary